sup3r/postprocessing/writers/base.py

```diff
@@ -105,6 +105,22 @@
         'min': -200,
         'max': 100,
     },
+    'temperature_min': {
```

Comment:
we should probably move this dict to its own file, huh?

Reply:
Yes could be json just need to make sure the file gets included in pip install like this: https://github.com/NREL/reV/blob/main/setup.py#L87

Want me to do this or have you done it already?

Reply:
Haven't done this yet so if you could that'd be awesome.

Reply:
Okay done, can you merge when happy?

Reply:
will do. thanks!